packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -206,7 +206,17 @@ Router = Router.extend({
           return this.showView(View);
         });
       } else {
-        this.createReactOrBackboneViewHandler(routeName, ConfirmSignupCodeView);
+        this.createReactOrBackboneViewHandler(
+          routeName,
+          ConfirmSignupCodeView,
+          {
+            ...Url.searchParams(this.window.location.search),
+            // for subplat redirect only
+            ...(this.relier.get('redirectTo') && {
```

Comment:
Sub plat seems to set `redirect_to` in the query params, do you need include

```
        // for subplat redirect only
        ...(this.relier.get('redirectTo') && {
          redirect_to: this.relier.get('redirectTo'),
        }),
```

Wouldn't it be covered with ```Url.searchParams(this.window.location.search)```

Reply:
The [SubscriptionsManagementRedirect](https://github.com/mozilla/fxa/blob/6eb5c55831a7c487d9c833625cf75e75799846e1/packages/fxa-content-server/app/scripts/views/subscriptions_management_redirect.js) wrapper around the subscriptions page has `mustAuth` set to true. Because of this, the `redirectTo` is set on the relier here when `/subscriptions` route is accessed without a valid session:
https://github.com/mozilla/fxa/blob/6eb5c55831a7c487d9c833625cf75e75799846e1/packages/fxa-content-server/app/scripts/views/base.js#L366

As far as I can tell, it's not typically added to query params.